test/conftest.py

```diff
@@ -57,3 +60,17 @@ def get_s3_key(url):
         return bucket.get_key(url.path)
 
     return get_s3_key
+
+
+@pytest.fixture
+def cli_tester(s3_bucket):
```

Comment:
`s3_bucket` not used? Or is it meant to be used in line 69:
`os.environ['DATALAKE_STORAGE_URL'] = s3_bucket`